acos_client/v21/axapi_http.py

```diff
@@ -122,6 +123,10 @@ def __init__(self, host, port=None, protocol="https", client=None, timeout=None)
                 self.port = 80
             else:
                 self.port = 443
+        if retry_errno_list is not None:
+            self.retry_errnos = retry_errno_list
+        else:
+            self.retry_errnos = [errno.ECONNRESET, errno.ECONNREFUSED]
```

Comment:
What if I want to add errors to the existing retry error list without knowing what the "bogosity in the API" is?